### DISCUSSION TITLE
libcudart.so.11.0

### DISCUSSION THREAD

**Discussion by hhzhao0525** [PaddlePaddle/PaddleOCR#13013] [Q&A]
请提供下述完整信息以便快速定位问题/Please provide the following information to quickly locate the problem

- 系统环境/System Environment：ubuntu16
- 版本号/Version：Paddle：2.4.2  PaddleOCR：2.7.0.3  问题相关组件/Related components：libcudart.so.11.0
- 运行指令/Command Code：python test.py, tets.py为官方事例代码
- 完整报错/Complete Error Message：Error: Can not import paddle core while this file exists: /root/anaconda3/envs/paddle_env/lib/python3.8/site-packages/paddle/fluid/libpaddle.so
Traceback (most recent call last):
  File "test.py", line 1, in <module>
    from paddleocr import PaddleOCR, draw_ocr
  File "/root/zhh6/PaddleOCR/paddleocr.py", line 21, in <module>
    import paddle
  File "/root/anaconda3/envs/paddle_env/lib/python3.8/site-packages/paddle/__init__.py", line 25, in <module>
    from .framework import monkey_patch_variable
  File "/root/anaconda3/envs/paddle_env/lib/python3.8/site-packages/paddle/framework/__init__.py", line 17, in <module>
    from . import random  # noqa: F401
  File "/root/anaconda3/envs/paddle_env/lib/python3.8/site-packages/paddle/framework/random.py", line 16, in <module>
    import paddle.fluid as fluid
  File "/root/anaconda3/envs/paddle_env/lib/python3.8/site-packages/paddle/fluid/__init__.py", line 36, in <module>
    from . import framework
  File "/root/anaconda3/envs/paddle_env/lib/python3.8/site-packages/paddle/fluid/framework.py", line 37, in <module>
    from . import core
  File "/root/anaconda3/envs/paddle_env/lib/python3.8/site-packages/paddle/fluid/core.py", line 338, in <module>
    raise e
  File "/root/anaconda3/envs/paddle_env/lib/python3.8/site-packages/paddle/fluid/core.py", line 274, in <module>
    from . import libpaddle
ImportError: libcudart.so.11.0: cannot open shared object file: No such file or directory

请尽量不要包含图片在问题中/Please try to not include the image in the issue.

**Comment by cuicheng01** [comment]
麻烦您提供下详细的环境信息呢？

**Comment by hhzhao0525** [comment]
> 麻烦您提供下详细的环境信息呢？

Package               Version
--------------------- -----------
anyio                 4.2.0
astor                 0.8.1
attrdict              2.0.1
Babel                 2.14.0
bce-python-sdk        0.9.4
blinker               1.7.0
cachetools            5.3.2
certifi               2024.2.2
charset-normalizer    3.3.2
click                 8.1.7
contourpy             1.1.1
cssselect             1.2.0
cssutils              2.9.0
cycler                0.12.1
Cython                3.0.8
decorator             5.1.1
et-xmlfile            1.1.0
exceptiongroup        1.2.0
Flask                 3.0.2
flask-babel           4.0.0
fonttools             4.47.2
future                0.18.3
h11                   0.14.0
httpcore              1.0.2
httpx                 0.26.0
idna                  3.6
imageio               2.33.1
imgaug                0.4.0
importlib-metadata    7.0.1
importlib-resources   6.1.1
itsdangerous          2.1.2
Jinja2                3.1.3
kiwisolver            1.4.5
lazy_loader           0.3
lmdb                  1.4.1
lxml                  5.1.0
MarkupSafe            2.1.5
matplotlib            3.7.4
networkx              3.1
numpy                 1.24.4
opencv-contrib-python 4.6.0.66
opencv-python         4.6.0.66
openpyxl              3.1.2
opt-einsum            3.3.0
packaging             23.2
paddlepaddle          2.6.0
pandas                2.0.3
pillow                10.2.0
pip                   24.0
premailer             3.10.0
protobuf              4.25.2
psutil                5.9.8
pyclipper             1.3.0.post5
pycryptodome          3.20.0
PyMuPDF               1.20.2
pyparsing             3.1.1
python-dateutil       2.8.2
pytz                  2024.1
PyWavelets            1.4.1
PyYAML                6.0.1
rapidfuzz             3.6.1
rarfile               4.1
requests              2.31.0
scikit-image          0.21.0
scipy                 1.10.1
setuptools            69.1.1
shapely               2.0.2
six                   1.16.0
sniffio               1.3.0
tifffile              2023.7.10
tqdm                  4.66.1
typing_extensions     4.9.0
tzdata                2023.4
urllib3               2.2.0
visualdl              2.5.3
Werkzeug              3.0.1
wheel                 0.42.0
zipp                  3.17.0

**Comment by tink2123** [comment]
可以尝试升级Paddle 2.5.2，或者找到 libcudart.so.11.0 所在的路径，添加至环境变量：

export LD_LIBRARY_PATH=$LD_LIBRARY_PATH:xxxx

**Comment by cuicheng01** [comment]
或者你可以尝试执行ldconfig
